.github/workflows/teamcity-pr-checks.yml

```diff
@@ -6,7 +6,7 @@ on:
   pull_request:
     paths:
       - '.github/workflows/teamcity-pr-checks.yml'
-      - 'mmv1/third_party/terraform/!.teamcity'
+      - 'mmv1/third_party/terraform/.teamcity'
```

Comment:
I thought that the `!` was required to use a folder starting with a dot but looks like I was wrong!